test/integration/framework/src/siftool/test_utils.py

```diff
@@ -347,12 +358,14 @@ def smart_contract_get_past_events(self, sc, event_name, from_block=None, to_blo
         finally:
             self.w3_conn.eth.uninstall_filter(filter.filter_id)
 
-    def tx_deploy_new_generic_erc20_token(self, deployer_addr, name, symbol, decimals):
+    def tx_deploy_new_generic_erc20_token(self, deployer_addr, name, symbol, decimals, cosmosDenom=None) -> Contract:
```

Comment:
params dont have type hints (i.e. deplayer_addr is this a string?) decimals is this an integer, bignumber or string?